src/types/transaction.rs

```diff
@@ -66,6 +66,10 @@ pub struct Receipt {
     /// Number of the block this transaction was included within.
     #[serde(rename = "blockNumber")]
     pub block_number: Option<U64>,
+    /// Sender
+    pub from: Address,
```

Comment:
I wonder if we should use `serde-default` here to maintain backward compatibility with clients without `from` field. Since it wasn't here for a while, I suspect that not all clients might have been returning the data
```suggestion
    #[serde(default)]
    pub from: Address,
```